fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/LoanScheduleModelDisbursementPeriod.java

```diff
@@ -138,4 +138,14 @@ public void setEMIFixedSpecificToInstallmentTrue() {
     public boolean isEMIFixedSpecificToInstallment() {
         return isEMIFixedSpecificToInstallment;
     }
+
+    @Override
+    public BigDecimal rescheduleInterestPortion() {
+        return null;
+    }
+
+    @Override
+    public void setRescheduleInterestPortion(BigDecimal rescheduleInterestPortion) {
+        return;
```

Comment:
as above

Reply:
There is one of the two implimentations of the interface "LoanScheduleModelPeriod" (Disbursement period and Repayment Period).  These methods are not significant for Disbursement Period and needed only in Repayment Period, hence they are marked dummy.